### DISCUSSION TITLE
Startpage on Safari missing margin

### DISCUSSION THREAD

**Discussion by ciruz** [community/community#9628] [Mobile] [answered]
Hey,

I noticed an incorrect margin bottom with Safari 15.2 (iPhone):

![image](https://user-images.githubusercontent.com/987947/148064176-324aa80e-718c-4e4b-823e-02a638e12260.png)

Same element with Chrome on Windows:

![image](https://user-images.githubusercontent.com/987947/148063931-dea94d53-0f36-46a0-b8c6-0c7745aa774a.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report — this appears to be an issue with our marketing pages and not the Mobile apps, but I'll pass this along to the team.

**Reply by ciruz** [reply]
Hey Brian, thanks. How could I report this to the right team, asking for the next time I find something.

**Reply by brianlovin** [reply]
The team is going to fix this soon by the way, thanks again for the report. I'm not actually sure the best way to report an issue with the site design, but in this case since you noticed it on mobile Safari, reporting it here ended up working out great 😊
